### ISSUE TITLE
Cannot focus docs with VoiceOver enabled

### ISSUE THREAD

**Issue by chrmarti** [Microsoft/vscode#27657]
- VSCode Version: Code - Insiders 1.13.0-insider (afe47dc84467b487ce0079ff72f55e7ef4f7ebca, 2017-05-30T10:34:18.253Z)
- OS Version: Darwin x64 15.6.0
- Extensions: none
---

Testing #27315

Steps to Reproduce:

1. Enable VoiceOver
2. Open content assist and try to pass focus to docs with Ctrl + Option + Space
3. -> VoiceOver starts reading the entire document

I think I'm using VoiceOver with its default settings.

**Comment by ramya-rao-a** [comment]
Why would one want to pass focus to docs while using screenreader?

Screenreader would read the docs if they are expanded when user moves up and down the list. 

The need to move focus to the docs is only for those who want to scroll down the docs but cannot use the mouse
